File: Frontend/src/app/models/HelperRejectedMail/helper-rejected-mail.ts
export class HelperRejectedMail {
  // tslint:disable-next-line:variable-name
  private email: string;
  // tslint:disable-next-line:variable-name
  private jbo: string;
  // tslint:disable-next-line:variable-name
  private poruka: string;
  constructor(jbo: string, email: string, poruka: string) {
    this.jbo = jbo;
    this.email = email;
    this.poruka = poruka;
  }

  get getJbo(): string {
    return this.jbo;
  }

  set setJbo(value: string) {
    this.jbo = value;
  }

  get getEmail(): string {
    return this.email;
  }

  set setEmail(value: string) {
    this.email = value;
  }

  get getPoruka(): string {
    return this.poruka;
  }

  set setPoruka(value: string) {
    this.poruka = value;
  }
}

File: Backend/src/main/java/isaps/tim18/PSW_ISA_ClinicalC_2019/model/HelperRejectedMail.java
package isaps.tim18.PSW_ISA_ClinicalC_2019.model;

public class HelperRejectedMail {
    private String email;
    private String jbo;
    private String poruka;

    public String getKorisnikovEmail() {
        return email;
    }

    public void setEmail(String email) {
        this.email = email;
    }

    public String getJbo() {
        return jbo;
    }

    public void setJbo(String jbo) {
        this.jbo = jbo;
    }

    public String getPoruka() {
        return poruka;
    }

    public void setPoruka(String message) {
        this.poruka = message;
    }
}
